jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -622,22 +665,29 @@ backend tcp-<%= tcp_proxy["name"] %>
     resolvers = "resolvers default "
   end
   backend_ssl = ""
-  if tcp_proxy["backend_ssl"] then
-    if tcp_proxy["backend_ssl"].downcase == "verify" then
+
+  if (tcp_proxy["backend_ssl"] || "").downcase != "verify"
```

Comment:
Though it is shorter, I find this actually harder to read than simply
```
if tcp_proxy["backend_ssl"] && tcp_proxy["backend_ssl"].downcase != "verify"
```

I favour clarity over compactness

Reply:
I think it would actually need to be

```
!tcp_proxy["backend_ssl"] || (tcp_proxy["backend_ssl"] && tcp_proxy["backend_ssl"].downcase != "verify")
```

Since we still need to abort for the case where `backend_ssl` is not provided

An alternative could be `if tcp_proxy.fetch("backend_ssl", "").downcase != "verify"` if you prefer?